bookclub-backend/models/books.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const booksSchema = new mongoose.Schema({
+  gid: String,
+  img: String,
+  title: String,
+  authors: [{ type: String }],
+  totalRating: Number,
+  reviews: [{ type: String }],
+})
+
+booksSchema.set('toJSON', {
+  transform: (doc, returnedObject) => {
+    returnedObject.id = returnedObject._id.toString();
```

Comment:
is there a reason why we're assigning a stringified version of `_id` and then removing `_id` from the database? We'd basically be removing the index of these files and replacing them with a string, which doesn't have the same effect.